pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -531,6 +544,15 @@ func NewOpenStackClient(cfg *AuthOpts, userAgent string, extraUserAgent ...strin
 		provider.HTTPClient.Transport = netutil.SetOldTransportDefaults(&http.Transport{TLSClientConfig: config})
 	}
 
+	if klog.V(6) {
+		provider.HTTPClient = http.Client{
+			Transport: &client.RoundTripper{
+				Rt:     &http.Transport{},
```

Comment:
Don't you want to reuse the info set in say line 544? (as provider.HTTPClient.Transport is completely overwritten)

Reply:
good point

Reply:
merged the fix I proposed in #789